model.py

```diff
@@ -5195,29 +5326,36 @@ def set_presentation_edition(self, policy):
         """
         _db = Session.object_session(self)
         old_presentation_edition = self.presentation_edition
-        all_editions = list(self.identifier.editions_in_priority_order)
+        all_editions = list(self.editions_in_priority_order())
         changed = False
+
+        from metadata_layer import (
```

Comment:
add TODO to come back later and see if can do a cleaner solution.
initially added import code in here to avoid circular import